riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/FailsafePluginBackupRequestTest.java

```diff
@@ -12,6 +12,8 @@
 import org.zalando.riptide.httpclient.ApacheClientHttpRequestFactory;
 
 import java.io.IOException;
+import java.time.Duration;
```

Comment:
Given that this imports where just added, but they are not used, the change looks unintended.

Reply:
Fixed